etc/migration/9.2-to-10.0/migration.cql

```diff
@@ -0,0 +1 @@
+ALTER "Principals" ADD "guest" boolean;
```

Comment:
We can also have "guest accounts" on regular tenants. Imagine:

Cam tenant with domain regex: .*.cam.ac.uk
GT tenant with domain regex: .*.gatech.edu

* Stuart (on the GT tenant) invites "foo@bar.cam.ac.uk"
* This causes a user account to be created on the *cam* tenant which is **private**

We need to know that user is still a "guest" so we can keep interacting (share more content by email) with them.

This reminds me that when we sign into a guest account for the first time we need to:
 * Change the user's visibility to the default tenant visibility
 * Remove the guest flag